drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -951,6 +955,54 @@ int bnxt_hwrm_vnic_rss_cfg(struct bnxt *bp,
 	return rc;
 }
 
+#define HW_MAX_VNICS 170
```

Comment:
Where did this number come from?  If we need to track it, we should be able to store what we used in func_cfg when we set up the VFs.